assets/src/components/task/style.scss

```diff
@@ -0,0 +1,88 @@
+.muriel-task {
+	border: 1px solid $muriel-gray-50;
+	display: flex;
+	padding: 24px 24px 24px 60px;
+	position: relative;
+
+  h1, h2 {
+  	max-width: 650px;
+  	padding: 0;
+  }
+
+	h1 {
+		color: $muriel-gray-800;
+		font-size: 24px;
+		font-weight: normal;
+    margin: 0 0 10px 0;
+	}
+
+	h2 {
+		color: $muriel-gray-600;
+		font-size: 15px;
+    font-weight: 200;
+    margin: 0;
+	}
+
+	.checklist__task-icon {
+		background: $muriel-white;
+		border: 2px solid $muriel-gray-200;
+    border-radius: 16px;
+		display: block;
+		height: 16px;
+		left: 24px;
+    position: absolute;
+    top: 24px;
+    width: 16px;
+	}
```

Comment:
I think you've mixed spaces and tabs; that's why the formatting is messed up in this file.

Reply:
There was a typo in the Prettier configuration file and a couple of differences in `.editorconfig` between this repo and https://github.com/Automattic/newspack-blocks. I fixed these in https://github.com/Automattic/newspack-plugin/pull/58/commits/3391656768aff8f011fcebd3fe57ea9cf8e28d80 and https://github.com/Automattic/newspack-plugin/pull/58/commits/c6759ad2a972789f35a3c74ff338f231f4679e57, then applied Prettier in https://github.com/Automattic/newspack-plugin/pull/58/commits/d39eae5021f81f996113093baaa41e091bc00548 and now I think these should be resolved.